### PR TITLE
Add product overview

### PR DESCRIPTION
**Pull request by chenette**

### CODE REVIEW

File: _includes/0.1/left_sidebar.html [hunk 1]
```diff
@@ -22,6 +22,10 @@
         <a href="{% link docs/0.1/index.md %}">Introduction</a>
         <a href="{% link docs/0.1/glossary.md %}">Glossary</a>
     </div>
+    <div class="left-sidebar-group">
+        <div class="left-sidebar-header">Grid Features</div>
```
Comment (vaporos) [inline _includes/0.1/left_sidebar.html:26]: Remove 'Grid' here.

File: _includes/0.1/left_sidebar.html [hunk 2]
```diff
@@ -22,6 +22,10 @@
...
+        <a href="{% link docs/0.1/grid_product.md %}">Grid Product</a>
```
Comment (vaporos) [inline _includes/0.1/left_sidebar.html:27]: And maybe also remove grid_ and Grid here.

File: docs/0.1/grid_product.md [hunk 3]
```diff
@@ -33,5 +33,7 @@ Grid Product can interact seamlessly with other smart contracts to manage owner
 and agent permissions,  product property templates (also called schemas),
 location data, and more.
 
+![Grid Product architecture](images/grid_product.png)
```
Comment (vaporos) [inline docs/0.1/grid_product.md:36]: Doesn't show (maybe forgot to add in the image?).

**Review by vaporos — CHANGES_REQUESTED** [review]

File: _includes/0.1/left_sidebar.html (hunk 2 above)
Comment (chenette) [reply _includes/0.1/left_sidebar.html]: I agree about removing "Grid" from the left-nav TOC, so I pushed a fixup commit with this change.

However, the file name should stay as is, per doc best practices (which recommend that the file name matches the topic title, which is "Grid Product").

**Review by vaporos — APPROVED** [review]

**Review by dplumb94 — APPROVED** [review]
Looks great!

File: docs/0.1/grid_product.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+creation.
+
+Grid Product can interact seamlessly with other smart contracts to manage owner
+and agent permissions,  product property templates (also called schemas),
+location data, and more.
+
+![Grid Product architecture](images/grid_product.png)
+
+These components combine to submit transactions to the back-end distributed
+ledger, securely and effectively sharing product data with all trading partners.
```
Comment (davececchi) [inline docs/0.1/grid_product.md:39]:
```suggestion
ledger, securely and efficiently sharing product data with all trading partners.
```

**Review by davececchi — APPROVED** [review]
This is super great.

One minor change suggested.

**Review by RyanLassigBanks — APPROVED** [review]

**Comment by chenette** [comment]
Thanks, @davececchi !  I'll make that change in the next update, which will be soon. Merging this initial version now, so there's a model for other overviews.

**Pull request merged**